service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupSpreadsheetRow.java

```diff
@@ -124,45 +127,43 @@ public class BackupSpreadsheetRow implements SpreadsheetRow {
   private String emergencyContacts;
 
   public BackupSpreadsheetRow(
-    LegacyBeacon legacyBeacon,
+    BeaconBackupItem legacyBeacon,
     ExportMapper exportMapper,
     DateTimeFormatter dateFormatter
   ) {
     BeaconExportDTO mappedLegacyBeacon = exportMapper.toLegacyBeaconExportDTO(
       legacyBeacon
     );
 
-    this.id = Objects.requireNonNull(legacyBeacon.getId()).unwrap();
+    this.id = Objects.requireNonNull(legacyBeacon.getId());
 
     setLegacyBeaconDetails(mappedLegacyBeacon, dateFormatter);
 
     setNotes(legacyBeacon.getData().getBeacon().getNote());
-    setUses(mappedLegacyBeacon.getUses());
-    setOwners(mappedLegacyBeacon.getOwners());
-    setEmergencyContacts(mappedLegacyBeacon.getEmergencyContacts());
+    setLegacyUses(mappedLegacyBeacon.getUses());
+    setLegacyOwners(mappedLegacyBeacon.getOwners());
+    setLegacyEmergencyContacts(mappedLegacyBeacon.getEmergencyContacts());
   }
 
   public BackupSpreadsheetRow(
     Registration registration,
-    AccountHolder accountHolder,
-    List<Note> notes,
-    ExportMapper exportMapper,
+    List<Note> nonSystemNotes,
+    BeaconUseMapper beaconUseMapper,
     DateTimeFormatter dateFormatter
-  ) {
-    BeaconExportDTO mappedBeacon = exportMapper.toBeaconExportDTO(
-      registration,
-      accountHolder,
-      notes
-    );
+  ) throws JsonProcessingException {
+    Beacon beacon = registration.getBeacon();
+    List<BeaconOwner> owners = registration.getBeaconOwner() != null
+      ? List.of(registration.getBeaconOwner())
+      : new ArrayList<>();
 
     this.id = registration.getBeacon().getId().unwrap();
 
-    setModernBeaconDetails(mappedBeacon, dateFormatter);
+    setModernBeaconDetails(beacon, dateFormatter);
 
-    setNotes(getStringifiedNotes(mappedBeacon.getNotes()));
-    setUses(mappedBeacon.getUses());
-    setOwners(mappedBeacon.getOwners());
-    setEmergencyContacts(mappedBeacon.getEmergencyContacts());
+    setNotes(getStringifiedNotes(nonSystemNotes));
+    setUses(registration.getBeaconUses(), beaconUseMapper);
+    setModernOwners(owners);
+    setModernEmergencyContacts(registration.getEmergencyContacts());
```

Comment:
as mentioned previously about the use of "set${VariableName}", maybe something more like populate or build